app/lib/account/backend.dart

```diff
@@ -538,7 +538,7 @@ class AccountBackend {
       return null;
     }
     return AuthenticatedUser(user,
-        audience: activeConfiguration.pubSiteAudience!);
+        audience: activeConfiguration.pubServerAudience!);
```

Comment:
This was sort of a bug, right? are there any implications?

Reply:
No implication, it was the same value.